ios/browser/api/bookmarks/brave_bookmarks_api.h

```diff
@@ -64,21 +63,22 @@ OBJC_EXPORT
 @property(nonatomic, readonly) bool isVisible;
 @property(nonatomic, readonly) bool isValid;
 
-@property(nonatomic, readonly) NSString* titleUrlNodeTitle;
-@property(nonatomic, nullable, readonly) NSURL* titleUrlNodeUrl;
+@property(nonatomic, copy) NSString* title;
+@property(nonatomic, nullable, copy) NSURL* url;
 
 @property(nonatomic, nullable, readonly) IOSBookmarkNode* parent;
 @property(nonatomic, readonly) NSArray<IOSBookmarkNode*>* children;
 @property(nonatomic, readonly) NSArray<BookmarkFolder*>* nestedChildFolders;
 @property(nonatomic, readonly) NSUInteger childCount;
 @property(nonatomic, readonly) NSUInteger totalCount;
+@property(nonatomic, readonly) NSUInteger childUrlCount;
+@property(nonatomic, readonly) NSUInteger totalUrlCount;
```

Comment:
These arent used anywhere

Reply:
Solved. It's from a bad rebase. It was supposed to replace `bookmarks(in: node).count` with just `node.childUrlCount`. 

That's fixed again.